static/docs/commands-reference/add.md

```diff
@@ -115,10 +115,7 @@ $ dvc add data.xml
 Adding 'data.xml' to '.gitignore'.
 Saving 'data.xml' to cache '.dvc/cache'.
 Saving information to 'data.xml.dvc'.
-
-To track the changes with git run:
```

Comment:
It might be beneficial to keep at least once per example? It might click for someone on how DVC works when they see this command.

Reply:
I thought they weren't relevant to the example so removed, but you're right. Restored in 6863597